src/components/Dropdown/Dropdown.module.css

```diff
@@ -63,3 +62,24 @@
 .sbui-dropdown__trigger {
   @apply border-none bg-transparent p-0 focus:ring-0;
 }
+
+.sbui-dropdown__arrow {
+  @apply fill-current text-bg-primary-light dark:text-bg-secondary-dark;
```

Comment:
I guess a problem we have here is that we have a border around the Dropdown, and the arrow would look great if the border went around the triangle arrow.
You could have dynamic borders here so that the arrow doesn't have a border completely surrounding it, and only went around the side you wanted it to.

I think you can do that by targeting the `data-side` attribute

```css
.sbui-dropdown__content[data-side="top"] .sbui-dropdown__arrow {
  /* 
    style the arrow here for top side
    you can do the same for each side
  */
}
```

Let me know what you think :D 

reference:
https://radix-ui.com/primitives/docs/components/dropdown-menu#collision-aware-animations

Reply:
I actually tried this out, but it turned out adding border top works all the time for all the side properties, so just added `border-t` it in the `.sbui-dropdown__arrow` instead

Reply:
looks great! thanks